docs/12-ParallelExecution.md

```diff
@@ -19,6 +19,47 @@ Docker works really well for isolating testing environments.
 By the time of writing this chapter, we didn't have an awesome tool like it. This chapter demonstrates how to manage parallel execution manually. As you will see we spend too much effort trying to isolate tests which Docker does for free. Today we **recommend using Docker** for parallel testing.
 </div>
 
+### Docker
+
+> Section is under construction
+
+Requirements:
+
+ - `docker` or [Docker Toolbox](https://www.docker.com/products/docker-toolbox)
+
+Build the `codeception/codeception` image
+
+    docker-compose build
+    
+Start 
```

Comment:
Here you should mention about docker compose and how to enable all services for it